tests/test_builders/test_build_linkcheck.py

```diff
@@ -1003,29 +1009,29 @@ def test_get_after_head_raises_connection_error(app):
     }
 
 
-@pytest.mark.sphinx('linkcheck', testroot='linkcheck-documents_exclude', freshenv=True)
+@pytest.mark.sphinx('linkcheck', testroot='linkcheck-documents_exclude')
 def test_linkcheck_exclude_documents(app):
     with http_server(DefaultsHandler):
         app.build()
 
     with open(app.outdir / 'output.json', encoding='utf-8') as fp:
         content = [json.loads(record) for record in fp]
 
-    assert content == [
-        {
-            'filename': 'broken_link.rst',
-            'lineno': 4,
-            'status': 'ignored',
-            'code': 0,
-            'uri': 'https://www.sphinx-doc.org/this-is-a-broken-link',
-            'info': 'broken_link matched ^broken_link$ from linkcheck_exclude_documents',
-        },
-        {
-            'filename': 'br0ken_link.rst',
-            'lineno': 4,
-            'status': 'ignored',
-            'code': 0,
-            'uri': 'https://www.sphinx-doc.org/this-is-another-broken-link',
-            'info': 'br0ken_link matched br[0-9]ken_link from linkcheck_exclude_documents',
-        },
-    ]
+    def sortfn(x: dict[str, str | int]) -> tuple[tuple[str, str], ...]:
```

Comment:
In a parallel execution, sometimes those are not sorted correctly. I'm not really sure whether it's an issue or not but I think it's because of how records are written on the disk.